FLHook.sln

```diff
@@ -68,25 +72,21 @@ Global
 		Plugins\PluginUtilities\PluginUtilities.vcxitems*{81a35b95-1ced-4f58-a24c-e2dc47d43dd0}*SharedItemsImports = 4
 		Plugins\PluginUtilities\PluginUtilities.vcxitems*{81d33b95-1ddd-4f58-a24c-e2ec4a143dd0}*SharedItemsImports = 4
 		Plugins\PluginUtilities\PluginUtilities.vcxitems*{81d35b95-1cdd-4f58-a24c-e2ec47d43dd0}*SharedItemsImports = 4
+		Plugins\PluginUtilities\PluginUtilities.vcxitems*{8cbf9ef0-324a-4699-8a26-c2999b99b225}*SharedItemsImports = 4

```

Comment:
It looks like we're adding more things than should be in this pull request to this file?